Source/WebCore/page/PageOverlay.cpp

```diff
@@ -94,10 +94,10 @@ IntRect PageOverlay::bounds() const
         int height = frameView->height();
 
         if (!ScrollbarTheme::theme().usesOverlayScrollbars()) {
-            if (frameView->verticalScrollbar())
-                width -= frameView->verticalScrollbar()->width();
-            if (frameView->horizontalScrollbar())
-                height -= frameView->horizontalScrollbar()->height();
+            if (RefPtr verticalScrollbar = frameView->verticalScrollbar())
```

Comment:
nit: In small scopes like this, we usually try to use smaller names, like `scrollbar` here.